asynch/pool.py

```diff
@@ -243,37 +279,42 @@ async def shutdown(self) -> None:
             self._closed = True
 
 
-async def create_pool(
-    minsize: int = constants.POOL_MIN_SIZE,
-    maxsize: int = constants.POOL_MAX_SIZE,
-    loop: Optional[asyncio.AbstractEventLoop] = None,
-    **kwargs,
-) -> Pool:
-    """Returns an initiated connection pool.
-
-    The initiated pool means it is filled with `minsize` connections.
-
-    Equivalent to:
-    1. pool = Pool(...)
-    2. await pool.startup()
-    3. return pool
-
-    Do not forget to `await pool.shutdown()` for resource clean-up.
-
-    :param minsize int: the minimum number of connections in the pool
-    :param maxsize int: the maximum number of connections in the pool
-    :param loop Optional[asyncio.AbstractEventLoop]: an event loop (asyncio.get_running_loop() by default)
-    :param kwargs dict: connection settings
-
-    :return: a connection pool object
-    :rtype: Pool
-    """
-
-    pool = Pool(
-        minsize=minsize,
-        maxsize=maxsize,
-        loop=loop,
-        **kwargs,
-    )
-    await pool.startup()
-    return pool
```

Comment:
The same rational described in the https://github.com/long2ice/asynch/pull/130/files#r2008997884 is applicable here.